test/e2e/fixture.go

```diff
@@ -90,12 +89,10 @@ func getFreePort() (int, error) {
 }
 
 func (f *Fixture) setup() error {
-	installer, err := install.NewInstaller(f.Config, install.InstallOptions{})
+	_, err := exec.Command("kubectl apply -f ../../manifests/.").Output()
```

Comment:
e2e tests require only CRDs, config maps/secrets

Reply:
@alexmt cool, was wondering if It was on to explicitly call out 

01a_application-crd.yaml and 01b_appproject-crd.yaml since these files names can change, but i guess I can keep them in a var so we can easily change them as needed so we are not loading in all the manifests.